tests/karate/src/test/java/be/rollCall/createRollCall.feature

```diff
@@ -0,0 +1,86 @@
+Feature: Create a RollCall
+  # TODO: Add missing test data
+  Background:
+    * call read('classpath:be/utils/server.feature')
+    * def createLAO = read('classpath:be/createLAO/create.feature@name=createLAO')
+    * def badMessageDataTest = read('classpath:be/common/badMessageDataRequest.feature')
+    * def valideRollCall = {}
```

Comment:
```suggestion
    * def validRollCall = {}
```

typo, you might want to fix that elsewhere as well

Reply:
Great thank you !